newton/sim/builder.py

```
@@ -2135,6 +2129,58 @@ def add_shape_sdf(
             key=key,
         )
 
+    def simplify_meshes(self):
+        import coacd
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)